src/main/java/org/jabref/gui/groups/GroupDialogView.java

```diff
@@ -67,15 +67,19 @@
     private final ControlsFxVisualizer validationVisualizer = new ControlsFxVisualizer();
     private final GroupDialogViewModel viewModel;
 
-    public GroupDialogView(DialogService dialogService, BibDatabaseContext currentDatabase, PreferencesService preferencesService, AbstractGroup editedGroup) {
-        viewModel = new GroupDialogViewModel(dialogService, currentDatabase, preferencesService, editedGroup);
+    public GroupDialogView(DialogService dialogService, BibDatabaseContext currentDatabase, PreferencesService preferencesService, AbstractGroup editedGroup, boolean isGroup) {
+        viewModel = new GroupDialogViewModel(dialogService, currentDatabase, preferencesService, editedGroup, isGroup);
 
         ViewLoader.view(this)
                   .load()
                   .setAsDialogPane(this);
 
         if (editedGroup == null) {
-            this.setTitle(Localization.lang("Add subgroup"));
+            if (isGroup == true) {
```

Comment:
```suggestion
            if (isGroup) {
```
No need to compare a boolean against "true"